### DISCUSSION TITLE
Image galleries and gallery links no longer downloading

### DISCUSSION THREAD

**Discussion by joeuser01** [aclap-dev/video-downloadhelper#192] [Help and Q&A]
### Your question

After the upgrade to 8.2, gallery and gallery link downloads are broken... files arent being saved to the specified directory and they arent being named according to the app settings, neither are they being saved to their respective sub-directories as specified in settings.

Initially, after the upgrade to 8.2, clicking the gallery link download option would cause firefox to pop up multiple windows/save dialogs, one for each image in the gallery, rather than the expected behavior of silently saving all the files in the background.

I didnt realize this 8.2 update had occurred, so I first tried uninstalling/re-installing the add-in, companion app and firefox to no avail.
Now, after re-installing, its the exact same behavior.

In firefox settings, if I manually specify a default download specific save location, the addin will save to the default FF save location as stated above, specified filenaming rules and sub-directories structures aren't being used 

Been using this app for several years without issue and now the situation is the same on multiple systems.

8.2 seems to be a regression.   Rolling back to a previous version if possible, at least until a patch is announced.

### Addon & CoApp version and details

version: 8.2.0.23
channel: stable
build date: 2024-02-14
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:123.0) Gecko/20100101 Firefox/123.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\admin",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by paulrouget** [comment]
Dupe of https://github.com/aclap-dev/video-downloadhelper/discussions/192
